src/components/pokedex.html

```diff
@@ -230,7 +230,7 @@ <h4 data-bind="text: `Showing ${PokedexHelper.filteredList().length} Pokémon`">
                                                 <img width="20px" data-bind="attr: { src: App.game.party.getPokemon($data.id).megaStone.getImage()}"/>
                                             </div>
                                             <!-- /ko -->
-                                            <img src="" width="80px" data-bind="css: { 'pokemon-not-seen': !PokedexHelper.pokemonSeen($data.id)(), 'pokemon-seen-but-not-caught': !App.game.party.alreadyCaughtPokemonByName(name) && PokedexHelper.pokemonSeen($data.id)() }, attr:{ src: PokemonHelper.getImage($data.id, App.game.party.alreadyCaughtPokemon($data.id, true) && !PokedexHelper.hideShinyImages())}">
+                                            <img src="" width="80px" data-bind="css: { 'pokemon-not-seen': !PokedexHelper.pokemonSeen($data.id)(), 'pokemon-seen-but-not-caught': !App.game.party.alreadyCaughtPokemonByName(name) && PokedexHelper.pokemonSeen($data.id)() }, attr:{ src: PokemonHelper.getImage($data.id)}">
```

Comment:
This breaks the Pokedex "Hide shiny images" toggle

Reply:
Yeah. I forgot what that code did. Readded it